src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,30 @@
-package mate.academy;
-
-public class Main {
-    public static void main(String[] args) {
-
-    }
-}
+package mate.academy;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
+public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
+
+    public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book book1 = new Book(1L, "Math", BigDecimal.valueOf(200));
+        Book book2 = new Book(2L, "Biology", BigDecimal.valueOf(250));
+        Book book3 = new Book(3L, "Chemistry", BigDecimal.valueOf(300));
+        bookDao.create(book1);
+        bookDao.create(book2);
+        bookDao.create(book3);
+        bookDao.deleteById(2L);
+        List<Book> books = bookDao.findAll();
+        Optional<Book> findBook = bookDao.findById(3L);

```

Comment:
it is a bad idea to pass ids because you can’t be sure that records with these ids exist in DB. better pass `book1.getId()` etc.